src/modules/missionHelper/i18n/nb_NO.json

```diff
@@ -60,34 +60,35 @@
 			"mobile_air_vehicles": "Røykdykkerbil | Røykdykkerbiler",
 			"mobile_command_vehicles": "Innsatsstøttebil | Innsatsstøttebiler",
 			"platform_trucks": "Stigebil | Stigebiler",
-			"police_cars": "Patruljebil | Patriljebiler",
+			"police_cars": "Patruljebil | Patruljebiler",
 			"police_helicopters": "Politihelikopter | Politihelikoptre",
 			"sheriff": "Utrykningspolitibil | Utrykningspolitibiler",
 			"swat_armored_vehicle": "Delta kjøretøy | Delta kjøretøy",
 			"swat_suv": "Panstret kjøretøy | Pansdrede kjøretøy",
-			"water_tankers": "Tankbil | Tankbiler"
+			"water_tankers": "Tankbil | Tankbiler",
+			"traffic_car": "Utrykningspolitibil"
```

Comment:
Do you want to add a pluralized value?

Reply:
I'll add this to the finished file 👍 thanks for the heads up